modelexpress_client/python/modelexpress/refit/reshard/verify.py

```
@@ -0,0 +1,154 @@
+
+    ``full_tensor`` is the flat staging buffer holding the whole logical source
+    tensor; a publisher's shard occupies the box at ``shard_offset`` with ``shape``.
+    Returned as a contiguous copy because :func:`tensor_digest` digests byte order
+    and the box is generally strided inside the full tensor.
+
+    Here rather than with the comparison gate because it is the inverse of what a
+    publisher digests: the publisher hashes its own contiguous shard, and this is how
+    a receiver recovers the same box out of the buffer it assembled.
+    """
+    view = full_tensor.reshape(tuple(global_shape))
+    for axis, (start, extent) in enumerate(zip(shard_offset, shape)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)